utils/docker/vcluster/daos-base/el8/Dockerfile

```
@@ -0,0 +1,92 @@
+	rm -f /etc/systemd/system/*.wants/* &&                                                     \
+	rm -f /lib/systemd/system/local-fs.target.wants/* &&                                       \
+	rm -f /lib/systemd/system/sockets.target.wants/*udev* &&                                   \
+	rm -f /lib/systemd/system/sockets.target.wants/*initctl* &&                                \
+	rm -f /lib/systemd/system/basic.target.wants/* &&                                          \
+	rm -f /lib/systemd/system/anaconda.target.wants/*
+STOPSIGNAL SIGRTMIN+3
+ENTRYPOINT [ "/sbin/init" ]
+
+# Base configuration of dnf
+RUN	dnf clean all &&                                                                           \
+	dnf makecache &&                                                                           \
+	dnf --assumeyes install dnf-plugins-core &&                                                \
+	dnf config-manager --save --setopt=assumeyes=True &&                                       \
+	dnf config-manager --save --setopt=fastestmirror=True &&                                   \
+	dnf config-manager --set-enabled powertools &&                                             \
+	dnf install epel-release
+
+# Update system
+RUN	dnf update
+
+# XXX NOTE XXX Variale allowing to build the image wihout using glolbal --no-cache option and thus
+# XXX NOTE XXX to not update all rpms.  To work properly a random value such as the date of the day
+# XXX NOTE XXX should be given.
+ARG	BUST_CACHE
+ARG	DAOS_REPOS="https://packages.daos.io/v2.0"
+ARG	DAOS_GPG_KEYS="https://packages.daos.io/RPM-GPG-KEY"
+ARG	DAOS_REPOS_NOAUTH=""
+RUN	[[ $BUST_CACHE ]] && dnf update ;                                                          \
+	for repo in ${DAOS_REPOS} ; do                                                             \
+		echo "[INFO] Adding rpm repository: $repo" &&                                      \
+		dnf config-manager --add-repo $repo ;                                              \
+	done &&                                                                                    \
+	for gpg_key in ${DAOS_GPG_KEYS} ; do                                                       \
+		echo "[INFO] Adding repositories gpg key: $gpg_key" &&                             \
+		rpmkeys --import $gpg_key ;                                                        \
+	done &&                                                                                    \
+	for repo in ${DAOS_REPOS_NOAUTH} ; do                                                      \
+		echo "[INFO] Disabling authentification for repository: $repo" &&                  \
+		dnf config-manager --save --setopt=${repo}.gpgcheck=0 ;                            \
+	done
+
+# Install daos package
+RUN	dnf install daos
+
+# Generate GPG authentification certificates for using DAOS authentification
+ARG	DAOS_AUTH=no
+RUN	if [[ "$DAOS_AUTH" == yes ]] ; then                                                        \
```

You shouldn't need the second set of quotes here at all?
```suggestion
RUN	if [ "$DAOS_AUTH" == yes ] ; then                                                        \
```

---

going to change it

---

fixed